README.md

```diff
@@ -150,12 +149,12 @@ Architecture styles define how different components of an application programmin
 
   Notifies systems when events occur
 
+<sub>⬆️ [_Back to Table of Contents_](#table-of-contents)</sub>
```

Comment:
We like the idea, but we'd like to explore other ideas with our team on how to execute on this.

We'd like to accept the rest of the PR, so if you could remove the "Back to Table of Contents" links and resolve the current conflicts with main, that would be great.

Thank you for contributing.

Reply:
Couldn't users just use the sticky TOC at the top left or right (depeding whether you use beta repo view or not)
![image](https://github.com/ByteByteGoHq/system-design-101/assets/81827001/f3d9243d-b420-4129-aa52-577e70dc37d9)

Reply:
> We like the idea, but we'd like to explore other ideas with our team on how to execute on this.
> 
> We'd like to accept the rest of the PR, so if you could remove the "Back to Table of Contents" links and resolve the current conflicts with main, that would be great.
> 
> Thank you for contributing.

Done!
Thank you for your attention.